app/sprinkles/account/locale/ar/validate.php

```diff
@@ -15,7 +15,8 @@
  * @author Abdullah Seba
  */
 return [
-    'VALIDATE' => [
-        'PASSWORD_MISMATCH' => 'يجب أن تكون كلمة المرور وكلمة المرور التأكيدية نفس',
```

Comment:
Not sure why the indentation changed here... Should be 4 spaces. Also not sure why php-cs-fixer can't fix that and StyleCI allows that...

Reply:
Fixed